update the faculty list of KAIST

minhkim opened the pull request:
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

_Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

**Inclusion criteria**

- [ ] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [ ] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [ ] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

minhkim committed:
update the faculty list of KAIST

emeryberger commented on csrankings.csv:
```diff
@@ -16116,6 +16120,7 @@ Sachit Rao,IIIT Bangalore,http://www.iiitb.ac.in/faculty_page.php?name=SachitRao
 Sadao Kurohashi,Kyoto University,http://nlp.ist.i.kyoto-u.ac.jp/member/kuro/index.html,gpKS5P0AAAAJ
 Sadia Sharmin,BUET,https://cse.buet.ac.bd/faculty_list/detail/sadia,NOSCHOLARPAGE
 Sadie Creese,University of Oxford,http://www.cs.ox.ac.uk/sadie.creese,NOSCHOLARPAGE
+Sae-Young Chung,KAIST,http://itml.kaist.ac.kr,k-o3JBIAAAAJ
```
Not here: https://cs.kaist.ac.kr/people/faculty

emeryberger commented on csrankings.csv:
```diff
@@ -13038,6 +13041,7 @@ Mukul S. Bansal,University of Connecticut,http://www.engr.uconn.edu/~mukul,XCa-z
 Mukulika Maity,IIIT Delhi,http://www.iiitd.ac.in/people/faculty,F5sooVMAAAAJ
 Muli Safra,Tel Aviv University,http://www.tau.ac.il/~safra,NOSCHOLARPAGE
 Mun Choon Chan,National University of Singapore,https://www.comp.nus.edu.sg/~chanmc,QVeujWMAAAAJ
+Munchurl Kim,KAIST,http://viclab.kaist.ac.kr,bGXte_4AAAAJ
```
Not listed in CS.
https://cs.kaist.ac.kr/people/faculty

emeryberger reviewed (changes requested):
Please remove non-CS faculty.

minhkim committed:
Merge branch 'gh-pages' of github.com:emeryberger/CSrankings into update-KAIST

minhkim committed:
update the KAIST faculty list

minhkim commented:
Thanks, emeryberger!

I just removed EE professors who work on the conjunction between CS and EE. Then, I just added two newly hired CS professors on exist in https://cs.kaist.ac.kr/people/faculty.

Cheers!

minhkim commented:
Thanks, Emery!

I found that I put my message to the old post. Here I copied my previous message in this window. I just removed EE professors who work on the conjunction between CS and EE. Then, I just added two newly hired CS professors on exist in https://cs.kaist.ac.kr/people/faculty.

Cheers!

Min

emeryberger commented on csrankings.csv:
```diff
@@ -18084,6 +18085,7 @@ Tadao Takaoka,University of Canterbury,http://www.cosc.canterbury.ac.nz/tad.taka
 Tadayoshi Kohno,University of Washington,http://homes.cs.washington.edu/~yoshi,s_YDrrgAAAAJ
 Tae Hyun Kim 0006,Hanyang University,https://sites.google.com/site/lliger9/home,NOSCHOLARPAGE
 Tae Oh,Rochester Institute of Technology,https://www.rit.edu/gccis/computingsecurity/people/tom-oh,ugbZoXwAAAAJ
+Tae-Kyun Kim,KAIST,https://sites.google.com/view/tkkim/,j2WcLecAAAAJ
```
This person says they are at Imperial (an adjunct appointment off campus doesn't qualify for inclusion). "Associate Professor (Reader) and the director of Computer Vision and Learning Lab at Imperial College London"

emeryberger replied:
Please remove and I will merge.

minhkim replied:
Dear Emery,
Tae-Kyun recently moved from ICL to KAIST. He is a "full-time" employee at KAIST and an "adjunct" professor at ICL. He was busy and thus was late in updating his website. Now he updated his website to reflect his changed status (https://sites.google.com/view/tkkim/). I do confirm that these two people are full-time faculties who are officially employed. Please merge these changes. Many thanks!
